### ISSUE TITLE
Why are these binaries needed?

### ISSUE THREAD

**Issue by ford04** [ScoopInstaller/Binary#10]
Could you make it a bit more transparent, what these binaries are and where/why they are needed in the first place? 

Thanks!

**Comment by niheaven** [comment]
> Scoops mirror for binaries that are no longer available

**Comment by ford04** [comment]
Yeah, I saw that hint. The emphasis is on a) providing information about the binaries and their purpose b) in which context they are used / which programs do need them in their installation routine - maybe the title was a bit misleading.

For example, I wondered, why python installation via scoop included binary "dark". There is no README, URL or similar provided [in the repo](https://github.com/ScoopInstaller/Binary/tree/master/dark) to clarify its usage. Given all these binaries in principle are a bunch of opaque `.exe` files, whose origin and intent is not clear, a bit more transparency would certainly make safety-conscious users more confident.

**Comment by r15ch13** [comment]
`dark` is provided by the https://wixtoolset.org/. Scoop requires it as an unpacker for some programs (like python) but the complete package is too big. So we extracted it and just distribute it this way.

All other programs in this repo are just mirrored because they don't have any official download and/or the webpage is down.
